molecule/cp-kafka-plain-rhel/molecule.yml

```diff
@@ -97,6 +105,4 @@ provisioner:
   inventory:
     group_vars:
       all:
-        sasl_protocol: plain
-
-        confluent_server_enabled: false
+        sasl_protocol: plain
```

Comment:
nit: newline

Reply:
Good catch, this will fail the linting check.  You can run linting locally by installing yamllint and then running something like the following in the root of the project before committing: yamllint -c .yamllint .

Reply:
Done. Don't see any other error/warning